CHANGES.rst

```diff
@@ -367,6 +369,8 @@ New Features
 
 - ``astropy.vo``
 
+  - KeyError when converting Table v1.2 numeric arrays fixed. [#4781]
```

Comment:
The workflow in your contribution howto says that you should edit the change file before doing the pull request. I thought that strange, but just used the issue number instead. I then linked that issue number with this pull request and vice versa. Seems the generating of issues, branches, and pull requests is still a bit murky. It would have been cleaner had I been able to (or knew how to?) tie this pull request to my issue. Oh well.

Now I am guessing my second email to remove the duplicate that I already had is really your @bsipocz request to change the PR. Funny, I am spending more time doing tedious changes than actually fixing code.